<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Отправка Датаграммы</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="le_lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pb_sendData">
         <property name="text">
          <string>Отправить Датаграмму</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pb_stop">
      <property name="text">
       <string>Остановить передачу</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTextEdit" name="te_result"/>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="pb_start">
      <property name="text">
       <string>Начать передачу</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lb_receiveData">
      <property name="text">
       <string>Принятые данные</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>te_result</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
